apps/designer/app/designer/features/style-panel/shared/get-final-value.ts

```diff
@@ -19,6 +19,12 @@ export const getFinalValue = ({
   if (currentValue?.value === "inherit" && inheritedValue !== undefined) {
     return inheritedValue;
   }
+  if (
+    currentValue?.type !== "unit" &&
+    isNumericString(String(currentValue?.value))
+  ) {
+    return { value: Number(currentValue?.value), type: "unit", unit: "number" };
```

Comment:
I see that the `unit` definition is `export const Unit = z.union([z.enum(units), z.literal("number")]);` so we have all possible `css` values and add our own "unit" "number",  probably better if it will have just `type="number"`
instead of type="unit" unit="number"
Otherwise for me it looks like mixing different concepts

Reply:
Not a single case, there are integer and number data types. Some are have sense only being positive etc
Probably its ok to be in unit.

`order`, `z-index` https://developer.mozilla.org/en-US/docs/Web/CSS/integer
`flex-grow` is [float >=0](https://developer.mozilla.org/en-US/docs/Web/CSS/number)